internal/controller/api/v1alpha1/host_types.go

```diff
@@ -17,25 +17,180 @@ limitations under the License.
+	IsMachineCidrDefined   host.ValidationStatus `json:"isMachineCidrDefined,omitempty"`
+	BelongsToMachineCidr   host.ValidationStatus `json:"belongsToMachineCidr,omitempty"`
+	IsApiVipConnected      host.ValidationStatus `json:"isApiVipConnected,omitempty"`
+	BelongsToMajorityGroup host.ValidationStatus `json:"belongsToMajorityGroup,omitempty"`
+	IsNtpSynced            host.ValidationStatus `json:"isNtpSynced,omitempty"`
+}
+
+type HostValidationInfo struct {
+	Hardware HardwareValidationInfo `json:"hardware,omitempty"`
+	Network  NetworkValidationInfo  `json:"network,omitempty"`
+}
+
+type HostProgressInfo struct {
+	CurrentStage   models.HostStage `json:"currentStage"`
+	ProgressInfo   string           `json:"progressInfo,omitempty"`
+	StageStartedAt string           `json:"stageStartedAt,omitempty"`
+	StageUpdatedAt string           `json:"stageUpdatedAt,omitempty"`
+}
+
+type L2Connectivity struct {
+	OutgoingIPAddress string `json:"outgoingIpAddress,omitempty"`
+	OutgoingNic       string `json:"outgoingNic,omitempty"`
+	RemoteIPAddress   string `json:"remoteIpAddress,omitempty"`
+	RemoteMac         string `json:"remoteMac,omitempty"`
+	Successful        bool   `json:"successful,omitempty"`
+}
+
+type L3Connectivity struct {
+	OutgoingNic     string `json:"outgoingNic,omitempty"`
+	RemoteIPAddress string `json:"remoteIpAddress,omitempty"`
+	Successful      bool   `json:"successful,omitempty"`
+}
+
+type HostConnectivityValidationInfo struct {
+	HostRef        *corev1.ObjectReference `json:"hostRef"`
+	L2Connectivity L2Connectivity          `json:"l2Connectivity,omitempty"`
+	L3Connectivity L3Connectivity          `json:"l3Connectivity,omitempty"`
+}
 
-	// Foo is an example field of Host. Edit Host_types.go to remove/update
-	Foo string `json:"foo,omitempty"`
+type HostNtpSources struct {
+	SourceName  string             `json:"sourceName,omitempty"`
+	SourceState models.SourceState `json:"sourceState,omitempty"`
 }
 
 // HostStatus defines the observed state of Host
 type HostStatus struct {
-	// INSERT ADDITIONAL STATUS FIELD - define observed state of cluster
-	// Important: Run "make" to regenerate code after modifying this file
+	State                 string                           `json:"state,omitempty"`
+	StateInfo             string                           `json:"stateInfo,omitempty"`
+	StateUpdatedAt        string                           `json:"stateUpdatedAt,omitempty"`
+	LogsCollectedAt       string                           `json:"logsCollectedAt,omitempty"`
+	InstallerVersion      string                           `json:"installerVersion,omitempty"`
+	UpdateTime            *metav1.Time                     `json:"updateTime,omitempty"`
+	CheckedInAt           *metav1.Time                     `json:"checkedInAt,omitempty"`
+	Hostname              string                           `json:"hostname,omitempty"`
+	Bootstrap             bool                             `json:"bootstrap,omitempty"`
```

Comment:
HasBootstrap

Reply:
@filanov @avishayt agree?

Reply:
Maybe `IsBootstrap` this flag indicate if this host will be the boostrap in the next installation attempt.

Reply:
Based on the [naming conventions](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions), prefixing with "has" or "is" is discouraged. I think leaving this as `bootable` and adding a comment explaining it (most or all of these fields should end up getting such comments) would be reasonable.

Reply:
It's not "bootable", but "bootstrap" - i.e., is this the agent that will run the bootstrap logic before it pivots

Reply:
So shall I leave it as is? `bootstrap`

Reply:
keeping `bootstrap` for now and marking as Resolved.
If you disagree, please make another comment and I'll change it.